src/instructlab/lab.py

```diff
@@ -593,6 +607,12 @@ def generate(
     default="",
     help="TLS client certificate password.",
 )
+@click.option(
+    "--model-family",
+    default="merlinite",
+    show_default=True,
+    help="model family to use when picking a chat template",
```

Comment:
```suggestion
    help="Model family to use when picking a chat template, e.g. 'merlinite', 'granite', etc.",
```

Reply:
I think this is a good thing to point out - though I would argue we can tackle this in a follow up PR. There is some ambiguity in how this help message is written because this parameter decides 2 different things. 
- which template to use for `generate`
- also which template to use for `chat` if a server is started in the background by `generate`

so we can think about structuring it a bit better and handle it in a follow up PR

Reply:
Right now it's very ambiguous to users, so definitely needs to be followed up on